src/main/java/com/github/technus/tectech/thing/casing/TT_Block_SpacetimeCompressionFieldGenerators.java

```diff
@@ -0,0 +1,9 @@
+package com.github.technus.tectech.thing.casing;
+
+import net.minecraft.block.Block;
+
+public class TT_Block_SpacetimeCompressionFieldGenerators {
```

Comment:
Why not group this and the other two also into `TT_Container_Casings`?

Reply:
Too many casings, this is also more organised since there are 9 per casing and 3 casings so 27 total. Trying to fill in the small gaps remaining is just not worth the effort. Also makes the multis code much more simple to work with.